doc/Makefile

```diff
@@ -17,4 +17,4 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
-	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O) -W
```

Comment:
I think warnings should be treated as errors: if you decide to write a docstring, the docstring syntax should be strictly correct

Reply:
Thats true, but I'm afraid I will have to do more fixing then